Resources/Prototypes/GameRules/events.yml

```diff
@@ -259,7 +259,7 @@
     startAudio:
       path: /Audio/Announcements/attention.ogg
     endAnnouncement: station-event-gas-leak-end-announcement
-    weight: 10
+    weight: 5 # CD: Nerf weight, please revert if upstream ever makes "dangerous" gasses more rare.
```

Comment:
I am not happy with decreasing the weight for the reasons we have talked about previously but I can't really think of a better solution other then making dangerous gasses more rare which I don't have the time to make an upstream PR for at the moment.